projects/uitdatabank/docs/entry-api/reuse-places-organizers.md

```diff
@@ -0,0 +1,231 @@
+
+Only in the case a certain place or organizer does not exist yet, a new one can be created.
+
+<!-- theme: warning -->
+
+> When sending event content to UiTdatabank trough the Entry API it is essential to reuse existing organizer IDs and place IDs. Otherwise your integration can be blocked because of the creation of duplicates.
+
+On this page, we outline some options to reuse existing places & organizers from UiTdatabank in your integration.
+
+## Reuse existing places
+
+### Storing the place ID(s) inside your application
+
+This is the simplest method. You store the identifier(s) of the place(s) inside your application.
+
+* ✅ Suitable when all the events created through your application all have the same location
+* ✅ Suitable when the events created through your application can have a different location, but (1) the amount of possible locations is limited & (2) the list of possible locations is static and won't change a lot over time
+* ❌ Not suitable when the locations of the events created through your application are unpredictable in advance
+
+#### Example
+
+In the example below, all the events created through the integration take place in "Het Depot", a concert hall in Leuven. First you'll need to find the identifier of the place `Het Depot` in UiTdatabank.
+
+**request**
+
+Perform a `GET` request to the UiTdatabank Search API, including the name of the place and the postal code:
+
+```
+GET /places/?q=name.\*:"Het Depot" AND address.\*.postalCode:3000
+```
+
+**response**
+
+The response now contains the identifier of "Het Depot":
+
+```js
+{
+   "@context":"http://www.w3.org/ns/hydra/context.jsonld",
+   "@type":"PagedCollection",
+   "itemsPerPage":30,
+   "totalItems":1,
+   "member":[
+      {
+         "@id":"https://io.uitdatabank.be/place/8248e289-c986-4006-902f-b0616dcbcde7",
+         "@type":"Place"
+      }
+   ]
+}
+```
+
+You can now store the identifier of the place "Het Depot" (`8248e289-c986-4006-902f-b0616dcbcde7`) in your application to reuse it as location ID in the `POST` request of the event(s) that you want to create.
+
+### Searching existing locations within the entry form
+
+In this solution you directly integrate with the UiTdatabank Search API in your own entry form.
+
+* ✅ Suitable when the locations for the events created through your application are unpredictable in advance
+* ❌ Not recommended when the events created through your application all have the same location
+* ❌ Not recommended when the events created through your application can have a different location, but (1) the amount of possible locations is limited & (2) the list of possible locations is static and won't change a lot over time
+
+#### Example
+
+In the example below, a user in your entry form types the name "Het Depot" in the input field for the location. Under the hood an API call to the UiTdatabank Search API is made to check if that place already exists:
```

Comment:
The example also filters on postal code, so we should mention that they also need to ask for the postal code first then?

Maybe we should provide an example screenshot / wireframe of such a UI?